### ISSUE TITLE
Pretty sure we're missing some db.close() statements.

### ISSUE THREAD

**Issue by tyler-hoffman** [power-cosmic/fog#30]
Just checked in with mongod, and after a few minutes of doodling around, mongo had 20 connections. Anyone else seeing this type of thing?

**Comment by pegurnee** [comment]
yeah, i was seeing that too, but i wasn't sure if that was because of me screwing around locally

**Comment by csperlich** [comment]
hmm, I'm not seeing that. How are you looking at it? I just ran mongostat, and doodled around for a bit and I couldn't get the conn field to increase pass 1

**Comment by csperlich** [comment]
If I look at the terminal that's actually running mongo, I see it get up to 5 connections at once, but then it goes back down immediately to 1 or 2, but maybe I'm not doing the correct sequence of actions.

**Comment by pegurnee** [comment]
what's the state of this?
